nni/algorithms/hpo/smac_tuner/convert_ss_to_scenario.py

```diff
@@ -189,7 +189,7 @@ def generate_scenario(ss_content):
         The scenario-object (smac.scenario.scenario.Scenario) is used to configure SMAC and can be constructed
         either by providing an actual scenario-object, or by specifing the options in a scenario file
     """
-    with open('scenario.txt', 'w') as sce_fd:
+    with open('/tmp/scenario.txt', 'w') as sce_fd:
```

Comment:
This change leads to the situation that smac cannot find 'scenario.txt', because for simplicity nni uses default parameters of the original smac which reads this file from the working directory. You can refer to this line for details: https://github.com/QuanluZhang/SMAC3/blob/master/smac/utils/io/cmd_reader.py#L36

Reply:
yes, I thought that this would be an easy fix, but it seems it involves a good amount of refactoring. Is there interest in fixing this? if yes, it would be interesting to know the steps so I can see if I have bandwidth. Otherwise, happy to close the PR

Reply:
Thanks for your contribution. The way is updating the package smac4nni to change the path accordingly. But this may induce version mismatch between smac4nni and nni. Thus, it would be better to keep it as is for now.